nessie/src/test/java/org/apache/iceberg/nessie/TestNessieCatalog.java

```diff
@@ -193,8 +205,16 @@ public void testWarehouseLocationWithTrailingSlash() {
 
   @Test
   @Override
-  @Disabled("Nessie currently returns an empty list instead of throwing a NoSuchNamespaceException")
+  @Disabled("Nessie doesnt create default namespaces")
+  public void testListNestedNamespaces() {
+    super.testListNestedNamespaces();
+  }
+
+  @Test
+  @Override
```

Comment:
you can now remove the overriden method from here so that the test from `CatalogTests` is executed